server/controllers/auth-controller.js

```diff
@@ -0,0 +1,40 @@
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const User = require("../models/user");
+const httpstatusCodes = require("../util/status-codes");
+const { validationResult } = require("express-validator");
+
+exports.postSignup = (req, res, next) => {
+  //##################### Error handler for invalid requests ##########################
```

Comment:
Do you want this particular comment to be removed or all the comments in this function?
To my understanding I think it will ease to understand the objective of this part of the code as most might not know about validationResult function.

Reply:
If feel we don't need any of these comments but it's up to you if you feel these comments increase your readability then you should keep them.